<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1169</width>
    <height>808</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Foucault2</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Foucault2.qrc">
    <normaloff>:/Foucault2/Foucault2.ico</normaloff>:/Foucault2/Foucault2.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="viewTimelines">
      <property name="dragMode">
       <enum>QGraphicsView::ScrollHandDrag</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1169</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionLoad"/>
    <addaction name="actionImport"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAdd_Task">
    <property name="title">
     <string>Tasks</string>
    </property>
    <addaction name="actionWork"/>
    <addaction name="actionAdd_comment"/>
    <addaction name="separator"/>
    <addaction name="actionNew_Couder_Screen"/>
    <addaction name="actionNew_Couder_Measure"/>
    <addaction name="separator"/>
    <addaction name="actionNew_Hartmann_Screen"/>
    <addaction name="actionAdd_Hartmann_Measure"/>
    <addaction name="separator"/>
    <addaction name="actionDiscard_last_task"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShow_mirror_both_side"/>
    <addaction name="actionShow_Colors"/>
    <addaction name="actionShow_smooth_curves"/>
    <addaction name="separator"/>
    <addaction name="actionDisplay_Compact"/>
    <addaction name="actionDisplay_Normal"/>
    <addaction name="actionDisplay_Details"/>
    <addaction name="actionDisplay_Full"/>
    <addaction name="separator"/>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAdd_Task"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionAdd_comment">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Comment</string>
   </property>
  </action>
  <action name="actionGlobal_settings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Global Settings</string>
   </property>
  </action>
  <action name="actionNew_Couder_Screen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Foucault Screen</string>
   </property>
  </action>
  <action name="actionNew_Couder_Measure">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Foucault Measure</string>
   </property>
  </action>
  <action name="actionNew_Hartmann_Screen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Hartmann Screen</string>
   </property>
  </action>
  <action name="actionAdd_Hartmann_Measure">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Hartmann Measure</string>
   </property>
  </action>
  <action name="actionAdd_Bath_Measure">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Bath Measure</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print...</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import...</string>
   </property>
  </action>
  <action name="actionShow_mirror_both_side">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show mirror both side</string>
   </property>
  </action>
  <action name="actionWork">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Work</string>
   </property>
  </action>
  <action name="actionShow_Colors">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Colors</string>
   </property>
  </action>
  <action name="actionDisplay_Compact">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Compact View</string>
   </property>
  </action>
  <action name="actionDisplay_Normal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Normal View</string>
   </property>
  </action>
  <action name="actionDisplay_Details">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Detailed View</string>
   </property>
  </action>
  <action name="actionShow_smooth_curves">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show smooth curves</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionDiscard_last_task">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete last task</string>
   </property>
  </action>
  <action name="actionDisplay_Full">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Full View</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Foucault2.qrc"/>
 </resources>
 <connections/>
</ui>
